apps/balancer-tools/src/app/apr/(utils)/tokenYield.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkChainId } from "@bleu-balancer-tools/utils";
+import { createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+
+import { rethRateProviderAbi } from "#/abis/rethRateProvider";
+import { blocks } from "#/lib/gql/server";
+
+export const publicClient = createPublicClient({
+  chain: mainnet,
+  transport: http(),
+});
+
+const SECONDS_IN_DAY = 86400;
+
+export async function getTokenYieldByPoolId() {
+  const { currentRate, rate24hAgo } = await getDailyRateDiff();
+
+  const calc = (currentRate - rate24hAgo) / rate24hAgo;
+
+  const apr = ((1 + calc) ** 365 - 1) * 100;
```

Comment:
redo this to something like 
```suggestion
  const apr = ((currentRate - rate24hAgo) / e18)* 365
```